pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -164,6 +165,11 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 					Resources: []string{"persistentvolumeclaims"},
 					Verbs:     []string{"get", "list", "watch"},
 				},
+				{
+					APIGroups: []string{policyv1beta1.GroupName},
```

Comment:
Just a note: this change was also added in #5624 so once the new etcd-druid is released, let's first merge that PR since it is smaller.